integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -1031,9 +1022,8 @@ object BloopDefaults {
           val config = {
             val c = Keys.classpathOptions.value
             val java = Config.Java(javacOptions)
-            val analysisOut = BloopKeys.bloopAnalysisOut.value.map(_.toPath)//.getOrElse(out.resolve(s"${projectName}-analysis.bin"))
             val compileSetup = Config.CompileSetup(compileOrder, c.bootLibrary, c.compiler, c.extra, c.autoBoot, c.filterLibrary)
-            val `scala` = Config.Scala(scalaOrg, scalaName, scalaVersion, scalacOptions, allScalaJars, analysisOut, Some(compileSetup))
+            val `scala` = Config.Scala(scalaOrg, scalaName, scalaVersion, scalacOptions, allScalaJars, None, Some(compileSetup))
```

Comment:
Is this `analysisOut` parameter used anywhere else? Otherwise we should maybe remove it.

```suggestion
            val `scala` = Config.Scala(scalaOrg, scalaName, scalaVersion, scalacOptions, allScalaJars, Some(compileSetup))
```

Reply:
It's used in some tests currently and it might be useful if we decide to take a stab at the offloading. Don't want to change the configuration right now (this is the same as the one used in `.bloop` files)